apis/apiextensions/v1/composition_revision_types.go

```diff
@@ -47,42 +47,10 @@ type CompositionRevisionSpec struct {
 	// Composition Functions, each of which is responsible for producing
 	// composed resources that Crossplane should create or update.
 	//
-	// "Resources" indicates that a Composition uses what is commonly referred
```

Comment:
Do we mean to remove stuff from v1 API? Seems like this will cause it to become backwards incompatible?

Reply:
Yes. This is called out in the v2 design and e.g. at https://docs.crossplane.io/v2.0-preview/whats-new/#backward-compatibility. This is "native patch and transform" composition (aka `mode: Resources` composition) that's being removed.

A bit of context:

* The feature has been marked deprecated and security-fix only for 3 releases (~9 months).
* It requires a significant amount of legacy code - there's a pretty big maintenance burden to keep supporting it
* There's a `function-patch-and-transform` that reimplements it as a function - so migration isn't terrible

With that in mind we feel good about taking the opportunity to drop the feature - even from a v1 API - in v2.0.